ChakraCore Samples/Hello World/Linux_OSX/sample.cpp

```diff
@@ -4,8 +4,22 @@
 //-------------------------------------------------------------------------------------------------------
 
 #include "ChakraCore.h"
+#include <stdlib.h>
```

Comment:
makes sense. We don't force devs to use c++11 though. We just use it internally.